src/index.ts

```diff
@@ -20,7 +20,7 @@ function processDryRun<T>(arg: T): T {
   }
 
   if (isDryRun()) {
-    logger.info('[dry-run] Dry-run mode is on!');
+    logger.debug('[dry-run] Dry-run mode is on!');
```

Comment:
This is very useful to see as a confirmation that the dry run flag was accepted, which makes quite a difference when running craft manually.

Reply:
Agreed, let's keep it at `info`.

Reply:
```suggestion
    logger.info('[dry-run] Dry-run mode is on!');
```